bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -361,13 +361,21 @@
 		<label>@text/channelButtonLabel</label>
 		<description>@text/channelButtonDescription</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
```

Comment:
autoUpdatePolicy = veto, so not sure we need a property

Reply:
Removed